ReactAndroid/src/main/java/com/facebook/react/devsupport/DevSupportManagerImpl.java

```diff
@@ -522,6 +532,15 @@ private void resetCurrentContext(@Nullable ReactContext reactContext) {
       mDebugOverlayController = new DebugOverlayController(reactContext);
     }
 
+    if (mDevSettings.isHotModuleReplacementEnabled() && mCurrentContext != null) {
+      try {
+        URL sourceUrl = new URL(getSourceUrl());
+        mCurrentContext.getJSModule(HMRClient.class).enable("android", sourceUrl.getPath().substring(1));
```

Comment:
Extract as a named local to document what `sourceUrl.getPath().substring(1)` represents